datahub-web-react/src/app/entity/shared/containers/profile/header/AddDeprecatedDataModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Button, DatePicker, Form, Input, message, Modal } from 'antd';
+import { useUpdateDeprecationMutation } from '../../../../../../graphql/mutations.generated';
+
+type Props = {
+    urn: string;
+    visible: boolean;
+    onClose: () => void;
+    refetch?: () => Promise<any>;
+};
+
+export const AddDeprecatedDataModal = ({ urn, visible, onClose, refetch }: Props) => {
+    const [updateDeprecation] = useUpdateDeprecationMutation();
+    const [form] = Form.useForm();
+
+    const handleClose = () => {
+        form.resetFields();
+        onClose();
+    };
+
+    const handleAdd = async (formData: any) => {
+        message.loading({ content: 'Updating...' });
+        try {
+            await updateDeprecation({
+                variables: {
+                    input: {
+                        urn,
+                        deprecated: true,
+                        note: formData.note,
+                        decommissionTime: formData.decommissionTime && formData.decommissionTime.unix(),
+                    },
+                },
+            });
+            message.destroy();
+            message.success({ content: 'Deprecation Updated', duration: 2 });
+        } catch (e: unknown) {
+            message.destroy();
+            if (e instanceof Error) {
+                message.error({ content: `Failed to update Deprecation: \n ${e.message || ''}`, duration: 2 });
+            }
+        }
+        refetch?.();
+        handleClose();
+    };
+
+    return (
+        <Modal
+            title="Add Deprecation Data"
```

Comment:
Minor: Let's change to "Deprecation Details"

Reply:
Done and Committed.